vm_supervisor/__init__.py

```diff
@@ -2,6 +2,10 @@
 from subprocess import check_output, CalledProcessError
 from typing import Optional
 
+from . import supervisor
+
+assert supervisor
```

Comment:
Why?

Reply:
To avoid unused variable warnings from flake8

Reply:
I guess this is related to the global app creation. Couldn't you use a create_app() function instead and call that from main()?

Reply:
If that's not feasible, add a comment.

Reply:
It is not required, it just makes it easier to import it from the package (`from vm_supervisor import supervisor, ...).

We should probably add it for most of the package, like [this FastAPI](https://github.com/tiangolo/fastapi/blob/12132276672c51412f3fc38bd5e228cf4a87ecfb/fastapi/__init__.py) `__init__.py` does.

Reply:
Oh yeah no issues but then we need to deactivate flake8 checks for imports like these. Fine for now, just add a comment though so that we don't forget why the assert is here.